app.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-classes-per-file */
+
+// Book Class: Represents a book
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  // Check if there is a local storage for books
+  static #checkStorage() {
+    return localStorage.getItem('books');
+  }
+
+  // Get all the books Infos from the local storage
+  static getBooks() {
+    let books = [];
+    if (Store.#checkStorage()) {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  // Save a book in local storage
+  static saveBook(book) {
+    const books = Store.getBooks();
+
+    books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  // Delete a book in local storage
+  static unSaveBook(title) {
+    const books = Store.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+// AppBook Class: Handle AppBook Tasks
+class AppBook {
+  // Display books to the screen
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => AppBook.addBookToList(book));
+  }
+
+  // Display one book to the screen
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list');
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+        <td>${book.title}</td>
+        <td>${book.author}</td>
+        <td><button type='submit' class='btn btn-outline-dark'>Remove</button></td>
+        `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('btn')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
```

Comment:
- [OPTIONAL] I would suggest that you use unique `id` property for each book since it would make it easier to remove books with duplicate values

- eg:

```javascript
  removeBook(bookID) {
    // bookID can be any other property, e.g. title or author, in as much is it unique
    this.books = this.books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
    return this.books; 
  }
```

Reply:
@BenMukebo, It's really great idea, the thing is I had some struggles on the way of **generating** these `ID's` so if you have any thoughts please let me know.